.nvmrc

```
@@ -0,0 +1 @@
+v22.15.0
```

to follow the agent version!

```suggestion
22.13
```